Bottom: 656c8c797c18df2868dfeb8c87cb06e5e157b176
Top:    f837e8076d388a6782fab930d73d90526582179c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 15:08:23 +0000

Refresh of turn-back-on-some-commented-3d

---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 2bb9bfd..52182c0 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -202,13 +202,11 @@ step_do_export (HID_Attr_Val * options)
         poly_DelContour (&curc);
       }
 
-#if 1
     mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
     mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
-#endif
 
     copper_layer_objects = object3d_from_copper_layers_within_area (piece);
     board_outline_list = g_list_concat (board_outline_list, copper_layer_objects);
